content/docs/typechecking-with-proptypes.md

```diff
@@ -1,18 +1,18 @@
 ---
 id: typechecking-with-proptypes
-title: Typechecking With PropTypes
+title: Checagem de tipos com PropTypes
```

Comment:
can we keep "Typechecking" in the translated version? it seems a very common term. something like the below, for example

```
Checagem de tipos (Typechecking) com PropTypes
```